src/Pages/Errors.php

```diff
@@ -59,7 +59,7 @@ public function tlms_showWarnings(): void
 
         if (! empty($this->talentlmsAdminErrors)) {
             foreach ($this->talentlmsAdminErrors as $message) {
-                echo '<div class="error notice is-dismissible">' . $message . '</div>';
+                echo '<div class="error notice is-dismissible">' . esc_html($message) . '</div>';
```

Comment:
We used it before, since we know that we are escaping the content before echo it, but they didn't like it. However if you use `esc_html` here then we will have this issue:
![Screenshot 2023-03-15 at 12 05 13](https://user-images.githubusercontent.com/95210555/225278730-aa372309-391a-4a3e-b5df-fc9ce68d78d2.png)
and we need something like this
![Screenshot 2023-03-15 at 12 05 36](https://user-images.githubusercontent.com/95210555/225278817-79f837e7-4dc5-47e5-a132-a49cac81b053.png)
I made a little investigation and if we use `$this->talentlmsAdminErrors[] = wp_kses($message, array('p'=>array(),'strong'=>array(), 'a'=>array('href'=>array())), array('http', 'https'));` on line 40, we will be ok..give it a try!

Reply:
wp_kses takes as arguments an array of html elements **to keep** so this solution will not have any difference on the end result. I have pushed a solution that works and we might not have issues from WP, by using wp_kses($message, array('strong'=>array()), array('http', 'https')) on the final echoed message. Check it out and let me know!